app/utils/dataHooks.ts

```diff
@@ -337,15 +340,13 @@ export function useStakeIncreaseUntil() {
 
     const { bs, cs, rs } = status;
 
-    if (isBlockSummaryV1(bs)) {
-        if (!isRewardStatusV1(rs)) {
-            throw new Error('Block summary and reward status do not match.'); // Should not happen, as this indicates rs and bs are queried with different blocks.
-        }
-
-        return getV1Cooldown(0, bs, cs, rs.nextPaydayTime, now);
+    if (!isBlockSummaryV1(bs) || !isRewardStatusV1(rs)) {
+        throw new Error(
+            'Block summary or reward status unexpectedly have version 0.'
```

Comment:
It is more future proof to state that they are not V1 here as we could have future versions, and then the error message would be incorrect.

Reply:
True